frontend/src/laws/EnergiemixDurchMarktGeregelt.ts

```diff
@@ -10,4 +10,12 @@ export default createLaw({
       stateDebt: -1000,
     }
   },
+
+  priority(game) {
+    const startYear = 2021 // TODO: Put constants in central place
+    const endYear = 2030 // TODO: Put constants in central place
```

Comment:
Hier sollten wir uns noch synchronisieren: ich habe bisher 2050 als Endedatum (für das Rating) verwendet.

Reply:
Ok. Wo kommt das bei Dir vor? Einige diese Infos stehen ja in `repository.ts`. Wäre das ok, die direkt hier zu importieren?

Und: Sind solche TODOs eigentlich noch hilfreich... Ich bin da etwas old-school und habe eigentlich keine guten Erfahrungen damit: Bleiben meist für immer drin... Kann man auf Issues im Code verweisen und die Stellen werden im Issue auch aungezeigt? Das wäre ja schicker: Dann würde ich für das Thema ein Issue anlegen.

Reply:
Ja, sicher, ein Import wäre gut - von woher auch immer, vielleicht auch in `repository.ts`. "Meine" 2050 kommt in `src/components/TimeIndicator.vue` vor. Ob wir nun 2050 oder 2030 machen, ist mir eigentlich wurscht, aber mit 2030 sind es halt leider nur wenige Spielrunden.

Hm, ich bin vermutlich noch viel old-schooliger. Im konkreten Fall ist jetzt wohl der Zeitpunkt gekommen, die Konstanten zu definieren - und kein Ticket anzulegen.